src/pykeen/nn/emb.py

```diff
@@ -929,6 +929,43 @@ def resolve_aggregation(
     return aggregation
 
 
+class AnchorSelection:
+    """Anchor entity selection strategy."""
+
+    def select(self, k: int, edge_index: np.ndarray) -> np.ndarray:
```

Comment:
if it's a single function class, why not use `__call__`

Reply:
925b0f2